<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatchedTracksPage</class>
 <widget class="QWidget" name="MatchedTracksPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>500</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="showLayout">
     <item>
      <widget class="QLabel" name="showLabel">
       <property name="text">
        <string>Show:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="matchedRadio">
       <property name="text">
        <string>Matched Tracks</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="uniqueRadio">
       <property name="toolTip">
        <string>Tracks that are unique to their sources</string>
       </property>
       <property name="text">
        <string>Unique tracks</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="excludedRadio">
       <property name="toolTip">
        <string>Tracks that have been excluded from synchronization due to ambiguity</string>
       </property>
       <property name="text">
        <string>Excluded tracks</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="showSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="filterLayout">
     <item>
      <widget class="QLineEdit" name="filterLine">
       <property name="placeholderText">
        <string>Filter tracks... (TODO)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="filterCombo">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeView">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="animated">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
